sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/ComplexTypes.scala

```diff
@@ -39,7 +39,18 @@ object SimplifyExtractValueOps extends Rule[LogicalPlan] {
       // Remove redundant field extraction.
       case GetStructField(createNamedStruct: CreateNamedStruct, ordinal, _) =>
         createNamedStruct.valExprs(ordinal)
-
+      case GetStructField(w@WithFields(struct, names, valExprs), ordinal, maybeName) =>
```

Comment:
minor, `w @ WithFields` spaces before and after `@`.

Reply:
might as well fix this while jenkins is down, done